.github/workflows/rust.yml

```diff
@@ -174,27 +207,31 @@ jobs:
           toolchain: stable
           override: true
 
+      - uses: Swatinem/rust-cache@v1
+
       - uses: actions-rs/cargo@v1
         with:
           command: build
           args: >
-            --features default,runtime-${{ matrix.runtime }}
+            --features default,sqlx-mysql,runtime-${{ matrix.runtime }}-${{ matrix.tls }}
 
       - uses: actions-rs/cargo@v1
         with:
           command: test
           args: >
-            --features default,sqlx-mysql,runtime-${{ matrix.runtime }}
+            --features default,sqlx-mysql,runtime-${{ matrix.runtime }}-${{ matrix.tls }}
 
   postgres:
     name: Postgres
     runs-on: ubuntu-20.04
+    needs: check
     env:
       DATABASE_URL: "postgres://root:root@localhost"
     strategy:
       matrix:
-        version: [13.3, 12.7, 11.12, 10.17, 9.6.22]
-        runtime: [async-std-native-tls]
+        version: [13.3, 12.7]
```

Comment:
We could not support old versions of Postgres?

Reply:
We can just thinking if it's too much loll

Reply:
Let me add it back then